x-pack/plugins/ml/public/maps/anomaly_source.tsx

```diff
@@ -43,7 +47,7 @@ export interface AnomalySourceDescriptor extends AbstractSourceDescriptor {
 
 export class AnomalySource implements IVectorSource {
   static mlResultsService: MlApiServices['results'];
-  static canGetJobs: boolean;
+  static mlLocator?: LocatorPublic<SerializableRecord>;
```

Comment:
This seems okay to leave as a static variable since it doesn't need to be replicated across instances.